gensim/sklearn_api/w2vmodel.py

```diff
@@ -44,23 +52,23 @@ def __init__(self, size=100, alpha=0.025, window=5, min_count=5, max_vocab_size=
         self.negative = negative
         self.cbow_mean = int(cbow_mean)
         self.hashfxn = hashfxn
-        self.iter = iter
+        self.epochs = epochs
         self.null_word = null_word
         self.trim_rule = trim_rule
         self.sorted_vocab = sorted_vocab
         self.batch_words = batch_words
-
+        
     def fit(self, X, y=None):
         """
         Fit the model according to the given training data.
         Calls gensim.models.Word2Vec
         """
         self.gensim_model = models.Word2Vec(
-            sentences=X, size=self.size, alpha=self.alpha,
+            sentences=X, size=self.vector_size, alpha=self.alpha,
```

Comment:
looks like you should change `size` to `vector_size` and same for `iter` here for avoid DeprecationWarning.